tcms/testcases/static/testcases/js/search.js

```diff
@@ -118,6 +118,37 @@ $(document).ready(function() {
 
     hookIntoPagination('#resultsTable', table);
 
+    const selectAllButton = $("#check-all")
+
+    selectAllButton.click(function () {
+        const rowCheckboxInputButton = $("input:checkbox[name='row-check']")
+        if (selectAllButton.prop("checked")) {
+            rowCheckboxInputButton.prop("checked", true);
+            table.rows().select()
+        } else {
+            rowCheckboxInputButton.prop("checked", false);
+            table.rows().deselect()
+        }
+    });
+
+    table.on('select', function (e, dt, type, indexes) {
+        if (type === 'row') {
+            const totalCheckBoxes = $("input:checkbox[name='row-check']").length;
+            const totalCheckedBoxes = $("input:checkbox[name='row-check']:checked").length;
```

Comment:
:dizzy_face: these variable names are making my head spin :dizzy:. With the small `ed` difference between the names it's just too much to try and even process. 

Make them more descriptive like `number of rows` or `total rows` vs `number of selected` / `selected rows`.